Workspace/src/ik/include/ik/Roboarm.h

```diff
@@ -7,11 +7,16 @@ class Roboarm {
     float velocityLink2;
     float velocityLink3;
 
-    float eeX;
-    float eeY;
+    float angleLink1;
+    float angleLink2;
+    float angleLink3;
+
+    float poseX [4];//ee,wrist,elbow//shoulde
```

Comment:
Maybe we can have a `Point` struct that contains an x and y fields. makes the code a bit cleaner